intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/idea/documentation/IdeaDeclarationDocumentationProvider.xtend

```diff
@@ -37,10 +37,10 @@ class IdeaDeclarationDocumentationProvider {
 
 	def protected getFileInfo(PsiEObject element) {
 		val resource = element.EObject.eResource
-		val module = workspaceCfgProvider.getWorkspaceConfig(resource.resourceSet).findProjectContaining(resource.URI)
-		if (module !== null) {
-			val uri = resource.URI.deresolve(module.path) 
-			return '''[�module.name�] �uri�'''
+		val projectConfig = projectConfigProvider.getProjectConfig(resource.resourceSet)
```

Comment:
Can we use com.intellij.openapi.project.ProjectUtilCore.displayUrlRelativeToProject(VirtualFile, String, Project, boolean, boolean) directly?